sources/packages/api/src/route-controllers/confirmation-of-enrollment/models/confirmation-of-enrollment.dto.ts

```diff
@@ -49,7 +49,18 @@ export class ConfirmationOfEnrollmentAPIInDTO {
 /**
  * Read only Interface for study break item.
  */
-export interface StudyBreak {
+export class StudyBreak {
```

Comment:
Please update comment to Class from "interface"

Reply:
Changed!